src/main/kotlin/com/reicheltp/celtic_rituals/init/ModBlocks.kt

```diff
@@ -10,7 +9,4 @@ object ModBlocks {
 
     @ObjectHolder("celtic_rituals:ritual_bowl")
     var RITUAL_BOWL: Block? = null
-
-    @ObjectHolder("celtic_rituals:ritual_bowl")
-    var RITUAL_BOWL_TILE: TileEntityType<*>? = null
```

Comment:
Why is this moved to another file? This belongs to the block, and TileEntity is not an Entity